doc/building.dox

```diff
@@ -644,8 +645,8 @@ replace `localhost:8080` with your server name, replace `update-job` with
     java -jar ~/jenkins-cli.jar -s http://localhost:8080 update-job Magnum-GLTests < package/ci/jenkins-gltests.xml
 
 Build is done using **Ninja** build system and everything possible is enabled,
-thus you need also **SDL2**, **GLUT** and **OpenAL** libraries. It expects
-that **GCC** >= 4.9, 4.8, 4.7 and **Clang** are installed and there
+thus you need also **SDL2**, **GLFW**, **GLUT** and **OpenAL** libraries. It
+expects that **GCC** >= 4.9, 4.8, 4.7 and **Clang** are installed and there
```

Comment:
This would imply that you have to update *all* packaging and CI scripts (which is, like, a thousand locations). Please revert this bit.

I'm not sure whether we want to require GLFW everywhere. I will think it through and do the required updates after merging.